src/test/java/api/requests/PickSlipsTests.java

```diff
@@ -228,7 +236,8 @@ void responseContainsPickSlipWithAllAvailableTokens() {
     assertThat(requesterContext.getString("city"), is(address.getCity()));
     assertThat(requesterContext.getString("region"), is(address.getRegion()));
     assertThat(requesterContext.getString("postalCode"), is(address.getPostalCode()));
-    assertThat(requesterContext.getString("countryId"), is(address.getCountryId()));
+    assertThat(requesterContext.getString("countryId"), is(address.getCountryId().isEmpty() ?
+      null : CountryCode.getByCode(address.getCountryId()).getName()));
```

Comment:
It looks like we're using the same class both in production code and in the test that is supposed to check code's correctness. If there was a bug in `CountryCode` and `getName()` returned an incorrect value for "US", this assert wouldn't catch the issue because we would have the same incorrect value both in the code and in the test. Should we rather compare `requesterContext.getString("countryId")` to a string defined in the tests (probably, as one of the parameters of the parametrized test)?

Reply:
done